packages/docs/docs/charting/index.md

```diff
@@ -14,6 +14,14 @@ Here is a **sample** of what a charting experience might look like - to be clear
 
 ![Chart sample](/img/tutorials/chart-sample.png)
 
+## Data Model
```

Comment:
Possible to make these images a bit bigger?  Hard to read in current config

Reply:
Unfortunately the width is already maxed out. Do you think I should go back to Awesomic and ask them to increase the text size?

Reply:
For the left-most column - I think yes.  Annotations are fine, could even be reduced a bit.  Send them the preview and they will get the picture I think